server/pulp/plugins/util/saxwriter.py

```diff
@@ -0,0 +1,94 @@
+from xml.sax.handler import ContentHandler
+from xml.sax.saxutils import escape, quoteattr
+
+
+class XMLWriter(ContentHandler):
+    """
+    XML writer similar to xml.sax.saxutils.XMLGenerator.
+    The result is indented XML, which is written sequentially to the stream.
+    """
+
+    def __init__(self, stream, encoding='utf-8', short_empty_elements=False):
+        ContentHandler.__init__(self)
+        self._write = stream.write
+        self._flush = stream.flush
+        self._encoding = encoding
+        self._short_empty_elements = short_empty_elements
+        self._pending_start_element = False
```

Comment:
A comment would be helpful about what exactly this boolean means. Looking at the code below, I'm trying to reverse-engineer it, but a comment would be easier.